src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  --heightInputBig: 70px;
+  --heightInputSmall: 42px;
+  font-weight: 400;
+}
+
+body {
+  padding: 0;
+}
+
+.search {
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-weight: 400;
+  line-height: normal;
+  box-sizing: border-box;
+}
+
+.search-big {
+  background-position: 25px;
+  background-size: 19px 19px;
+  padding-left: 62px;
+  font-size: 16px;
+  height: var(--heightInputBig);
+  font-weight: 400;
+}
+
+.search-small {
+  background-position: 12px 14px;
+  background-size: 11px 11px;
+  height: var(--heightInputSmall);
+  padding-left: 33px;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.searchForm {
+  margin: 20px 0;
+}
+
+.search:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+}
+
+.searchForm:focus-within .search {
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-family: "Avenir-Heavy", sans-serif;
```

Comment:
If you use same font at all project, you should move it to very top of you html block tree, so thats meaning to body selector or to html selector

Reply:
At first,i did that,but for some reason it didn't work(

Reply:
html {
  font-family: "Avenir", "Montserrat", sans-serif;
  } -it doesn't work
  
  .search {
  font-family: "Avenir", "Montserrat", sans-serif;} -but it does